elixir/apps/domain/lib/domain/mailer/notifications.ex

```diff
@@ -2,17 +2,14 @@ defmodule Domain.Mailer.Notifications do
   import Swoosh.Email
   import Domain.Mailer
   import Phoenix.Template, only: [embed_templates: 2]
-  import Phoenix.VerifiedRoutes
-
-  @endpoint Web.Endpoint
```

Comment:
This is actually causing an issue on staging/prod where this module is not available due to the `domain` app not having `web` as a dependency. Outdated gateway emails are currently broken until this PR goes in.